doc/developers_guide/developers_guide.md

```diff
@@ -37,6 +32,33 @@ mvn install --projects . -DskipTests
 
 After that the dependencies of PK are available in your local maven repository in the version of the current release and hence references to these versions in the pom of the current release are valid.
 
+## Running Manual Tests To Examine a Specific Project
+
+In case on of the users of PK reports a bug for a specific project you can reproduce and debug PK's behavior in your IDE by using an appropriate run configuration.
```

Comment:
```suggestion
In case one of the users of PK reports a bug for a specific project you can reproduce and debug PK's behavior in your IDE by using an appropriate run configuration.
```